benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -554,6 +554,22 @@ class KeycloakScenarioBuilder {
     this
   }
 
+  def getClientUUID(): KeycloakScenarioBuilder = {
+    chainBuilder = chainBuilder
+      .exec(http("List Clients and get a Client UUID")
+        .get(ADMIN_ENDPOINT + "/clients")
+        .header("Authorization", "Bearer ${token}")
+        .queryParam("first",0)
+        .queryParam("max", 50)
+        .check(
+          status.is(200))
+        //ToDo Randomize the index of the client object to not fetch skewed Session stats
+        .check(jsonPath("$[10].id").notNull.saveAs("clientUUID"))
```

Comment:
Instead of keeping the TODO, I'd like to use the existing randomization for clients. I lookup the URL parameters in the Keycloak code; they will need some testing. If it doesn't work in a straighforward way, I'd be happy to take another look.

```suggestion
        .get(ADMIN_ENDPOINT + "/clients?clientId=${clientId}")
        .header("Authorization", "Bearer ${token}")
        .queryParam("first",0)
        .queryParam("max", 1)
        .check(
          status.is(200))
        .check(jsonPath("$[0].id").notNull.saveAs("clientUUID"))
```